app/d3plus.css

```diff
@@ -7,35 +7,77 @@
 
 .d3plus-tooltip {
   box-shadow: rgba(0, 0, 0, 0.25) 0px 1px 3px;
-  max-width: 225px;
-  min-width: 150px;
+  max-width: 224px;
+  min-width: 224px;
   z-index: 30;
+  padding: 0px !important;
+  border-radius: 6px !important;
+  & .d3plus-tooltip-title {
+    min-width: 224px;
+    background: var(--tooltip-background);
+    border-radius: 6px 6px 0px 0px;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-bottom: 7px;
+    padding-top: 10px;
+    & .icon {
+      position: fixed;
+      top: -21px;
+      right: 91px;
+      left: 91px;
+      width: 42px;
+      height: 42px;
+      border-radius: 50%;
+      padding-top: 5px;
+      & img {
+        height: 30px;
+      }
+    }
+    & .title {
+      padding-top: 15px;
+    }
+    & .title-without-icon {
+      padding-top: 0px;
+    }
+    & .subtitle {
+      padding-top: 4px;
+      color: var(--tooltip-subtitle);
+      font-size: 8px;
+      font-weight: 500px;
+      text-transform: uppercase;
+    }
+  }
   & .d3plus-tooltip-table {
-    border-collapse: collapse;
+    border-collapse: unset !important;
+    padding-left: 19px;
+    padding-right: 24px;
+    padding-top: 14px;
+    padding-bottom: 9px;
     & .d3plus-tooltip-tbody {
       & tr {
-        border-bottom: 1px solid #ddd;
-        font-family: "Palanquin", sans-serif;
-        line-height: 14px;
-        &:last-child {
-          border-bottom: none;
-        }
+        font-family: "Palanquin";
+        border: none !important;
         & td {
-          padding: 3px;
           text-align: left;
+          font-weight: 600;
+          padding-bottom: 7px;
           &:last-child {
             text-align: right;
           }
-          &:only-child {
-            font-weight: bold;
-            padding-top: 6px;
-            text-align: left;
-          }
         }
       }
     }
   }
   & .d3plus-tooltip-footer {
+    background: white;
     text-align: center;
+    padding-top: 10px;
+    padding-bottom: 16px;
+    border-radius: 0px 0px 6px 6px;
+    border-top: 1px solid var(--light-3);
+    //background-image: url("/icons/tooltip/template.svg");
+    //background-repeat: no-repeat;
+    //background-position: 20% 40%;
+    //background-color: transparent;
```

Comment:
What's the deal with this `template.svg`? Are we using it anywhere?

Reply:
that image goes in the footer of the home tooltip, I added it to the home and when I checked the profiles I commented it out because I didn't like how it looked :( 

is there a way to add the image only in 1 tooltip?

<img width="243" alt="Captura de Pantalla 2022-11-15 a la(s) 17 00 48" src="https://user-images.githubusercontent.com/59574973/202014176-46d1a172-6623-4e10-b2af-dd8ce20db1e2.png">

<img width="414" alt="Captura de Pantalla 2022-11-15 a la(s) 17 05 54" src="https://user-images.githubusercontent.com/59574973/202015049-919c2b2f-2c5b-4777-b4ec-1f91fbf95997.png">

Reply:
I _believe_ you can actually pass HTML in the string you provide `tooltipConfig.footer`, so maybe remove it from the CSS and try something like this in the homepage config:

```js
  footer: `<img src="/icons/tooltip/template.svg" /> Click to View Report`,
```

(you might need some additional attributes/styles, like `width` and `height`)